src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/SessionPass.php

```diff
@@ -22,16 +22,29 @@ class SessionPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition('session')) {
+        if (!$container->has('session')) {
             return;
         }
 
+        // BC layer: When user overrides the `session` service it's not an alias anymore.
+        if ($container->hasDefinition('session')) {
```

Comment:
there could be a chance the user defined an alias as well right ... :sweat_smile:

Reply:
fixed by removing session from config